### DISCUSSION TITLE
Better Notes - is better than I could ever imagine

### DISCUSSION THREAD

**Discussion by jparker1455** [windingwind/zotero-better-notes#1214] [General]
@windingwind My goodness, I have just been playing around with Better Notes for the last day or so.  I finally got Zotero to work for me and upload all my files.  I can not express how amazing your plugin is.  I was using Obsidian, which is great, but the ability to work inside Zotero exclusively, and drag and drop annotations in to Stand alone notes is just pure genious.  

There is very little in the way of good instructions in English to really explain how it works.  I did find this https://zotero.yuque.com/staff-gkhviy/better-notes/qy4cpe which is great but it is a real pain to have the browser translate.  If I cut and paste the translation it doesn't carry over and winds up being in Chinese.  

Anyway, I am willing to offer a collaboration to create an English version of how to use Better Notes.  I think you will be able to reach a larger audience.  

Having said all of that...nevertheless, Better Notes actually leveled up Zotero in such a huge way.  I don't even think I need Obisidian anymore for a way to make sense of all my literature and make the connections.  I develop a lot of content for my clients and it has been such a pain to update literature quotes, keep track of what I have read, make updates to content on new ideas and new connections.  Zotero would benefit from actually just integrating it into Zotero.  I hope they do and I hope that it would be a good thing for you as well.

I can not thank you enough for your efforts.

**Comment by windingwind** [comment]
Thank you for your feedback! I truly value contributions to the community, whether through coding, writing documentation, or answering questions. Unfortunately, I haven’t had the time to create a detailed document, and the one you mentioned is a bit outdated. If you're interested, it would be fantastic and incredibly helpful if you could help write updated documentation for this plugin. Your support would make a big difference!

**Comment by jparker1455** [comment]
:)  I will do my best.  Is there any way to get english captions on this video? https://www.bilibili.com/video/BV1Cv4y1M7BY/?spm_id_from=888.80997.embed_other.whitelist&t=19.993549&bvid=BV1Cv4y1M7BY&vd_source=cec958f17e5c8a167b44ddb44246b581

I tried a bunch of ways and I can't seem to get it setup.

Also, can you confirm that "workspaces" are no longer a thing.  I also understand that Master Notes are no longer limited to just one.

**Reply by windingwind** [reply]
Oh, that video is also super outdated. It's based on an early version of the plugin, so... not sure if it's still a good reference. I am also not sure if there is a way to get english captions. maybe not.

> Also, can you confirm that "workspaces" are no longer a thing. I also understand that Master Notes are no longer limited to just one.

Yes, it's no longer a thing. Can just ignore the concept of the master note, as when you have multiple _master_ notes, they are all but just normal notes. With the link creator, you can link to/from any note.

**Reply by windingwind** [reply]
The video is outdated, but most of its content is covered in the document you mentioned. I would say that only the examples of best practices are still useful: https://zotero.yuque.com/staff-gkhviy/better-notes/dko1mg
